addons/ui/org.openhab.ui.cometvisu.php/META-INF/MANIFEST.MF

```diff
@@ -17,7 +17,7 @@ Import-Package:
  com.caucho.vfs;version="[4.0,5)",
  javax.servlet;version="[3.1,4)",
  javax.servlet.http;version="[3.1,4)",
+ org.openhab.ui.cometvisu.php,
```

Comment:
Actually, this is not a package from this bundle (although the names are the same). It is an exported package from `org.openhab.ui.cometvisu` bundle.